fendermint/vm/interpreter/src/fvm/exec.rs

```diff
@@ -252,7 +254,17 @@ where
             PowerUpdates::default()
         };
 
-        let ret = (updates, next_gas_market.block_gas_limit);
+        let current_validators = self
+            .gateway
+            .current_membership(&mut state)
+            .context("failed to update membership")?
```

Comment:
this comment should be read membership, not update.

Reply:
Yeah, I have noticed it. Fixed